jobs/async-upload/pyproject.toml

```diff
@@ -10,11 +10,16 @@ packages = [{include = "job"}]
 python = ">=3.9,<4.0"
 configargparse = ">=1.7.1,<2.0.0"
 model-registry = { version = ">=0.2.19,<0.3.0", extras = ["boto3", "olot"] }
+requests = "^2.31.0"
+pyyaml = "^6.0"
```

Comment:
shoudln't these 2 be dev/integraton dependencies too? 🤔

Reply:
this was moved to the correct section, whoops!